homeassistant/components/tplink/__init__.py

```diff
@@ -102,7 +103,20 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
             f"Unexpected device found at {host}; expected {entry.unique_id}, found {found_mac}"
         )
 
-    hass.data[DOMAIN][entry.entry_id] = TPLinkDataUpdateCoordinator(hass, device)
+    parent_coordinator = TPLinkDataUpdateCoordinator(hass, device, timedelta(seconds=5))
+    child_coordinators: list[TPLinkDataUpdateCoordinator] = []
+
+    if device.is_strip:
+        child_coordinators = [
+            # The child coordinators only update energy data so we can
+            # set a longer update interval to avoid flooding the device
+            TPLinkDataUpdateCoordinator(hass, child, timedelta(seconds=60))
```

Comment:
There was still an outstanding discussion in the original PR https://github.com/home-assistant/core/pull/103668#discussion_r1387340343